modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BJSON.java

```diff
@@ -290,24 +291,24 @@ public BIterator newIterator() {
             if (isJSONArray) {
                 long cursor = this.cursor++;
                 if (arity == 1) {
-                    return new BValue[]{new BJSON(collection.value.get((int) cursor))};
+                    return new BValue[] {new BJSON(collection.value.get((int) cursor))};
                 } else {
-                    return new BValue[]{new BInteger(cursor), new BJSON(collection.value.get((int) cursor))};
+                    return new BValue[] {new BInteger(cursor), new BJSON(collection.value.get((int) cursor))};
                 }
             }
             if (arity == 1) {
-                return new BValue[]{new BJSON(collection.value.get((String) iterator.next()))};
+                return new BValue[] {new BJSON(iterator.next().getValue())};
```

Comment:
You've to have another space after "{" :) ..

Reply:
@lafernando: I don't think, it is not required. Even in the check style/google code style sample, that space is not there. 

http://checkstyle.sourceforge.net/config_coding.html#ArrayTrailingComma
https://google.github.io/styleguide/javaguide.html#s4.8.3-arrays